### ISSUE TITLE
Encountering errors while trying to run examples geotiff examples

### ISSUE THREAD

**Issue by lzachmann** [GoogleCloudPlatform/dataflow-geobeam#44]
**tl;dr** This may be an issue with my environment (I'm running Python 3.9.13), but I've had no success getting any of the examples involving gridded data (e.g., `geobeam.examples.geotiff_dem`) to run locally. Have these been tested recently? 

I was bumping up against `TypeError: only size-1 arrays can be converted to Python scalars [while running 'ElevToCentimeters']`, which were fixed by using `x.astype(int)` where appropriate. Then I hit `TypeError: format_record() takes from 1 to 2 positional arguments but 3 were given [while running 'FormatRecords']`. (Maybe [this line](https://github.com/GoogleCloudPlatform/dataflow-geobeam/blob/main/geobeam/examples/geotiff_dem.py#L52) needs to be something like `'FormatRecords' >> beam.Map(format_rasterpolygon_record, 'int', known_args.band_column)` instead?) Then I got `TypeError: Object of type 'ndarray' is not JSON serializable [while running 'WriteToBigQuery/BigQueryBatchFileLoads/ParDo(WriteRecordsToFile)/ParDo(WriteRecordsToFile)']` and stopped to jump in here. If it's just my environment, then I'm making changes needlessly. If it's the code, it seemed better for these fixed to be applied directly in the upstream repo. 

Any advice you have would be much appreciated!!

**Comment by tjwebb** [comment]
Hey @lzachmann, thanks for bringing this to my attention. I will take a look today and report back.
